### PR TITLE
Generate nuget packages in AppVeyor builds

### PR DESCRIPTION
**Pull request by andresmoschini**
These changes allow generating nuget packages in AppVeyor builds, each nuget package will have his version, for example _0.0.1-alpha1-build000006_

### CODE REVIEW

**Comment by andresmoschini** [comment]
Hi, it is me again @MakingSense/never-pony ! 

I added some usage details to the readme file: https://github.com/andresmoschini/doppler-feature-toggle/blob/0827d87240185ca9fc00dcb98d5f1cd1bde11eef/README.md

File: appveyor.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 version: 0.0.1-alpha1-{build}
 image: Visual Studio 2017
```
Comment (cristiangiagante) [inline appveyor.yml]: This image name will be the final name? Only like a suggerence we can name the image with some information about the environment or task.

**Comment by cristiangiagante** [comment]
I like it a lot 👍  great work!

File: appveyor.yml (hunk 1 above)
Comment (andresmoschini) [reply appveyor.yml]: I thought that image is related to the image used by AppVeyor.

**Pull request merged**